cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -98,6 +98,9 @@ function deploy_cdi() {
 function deploy_local_storage() {
     if [ "$KUBEVIRT_DEPLOY_LOCAL_STORAGE" == "true" ]; then
         $kubectl create -f /provision/local-volume.yaml
+        if [ "$KUBEVIRT_STORAGE" == "rook-ceph-default" ] || [[ "$KUBEVIRT_PROVIDER_EXTRA_ARGS" =~ enable-ceph ]]; then
```

Comment:
Perhaps this should be formulated as:
- Don't set as default storage class on the YAML (neither true nor false)
- If there is no other default storage class by this point, set local as default storage class.
To avoid hard-coding the logic of ceph

Reply:
pushed change that does that + fixes the [failure](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/860/check-provision-k8s-1.24/1561257325266735104) (will order commits once it passes)

Reply:
can use:
```
default_storage_class = $($kubectl get storageclass -o jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io/is-default-class=="true")].metadata.name}')
if [ -z $default_storage_class ];
```
to test if there is no default storage class, instead of having logic specific to ceph

Reply:
nice done

for some reason there is a failure here
not sure why yet

https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/860/check-provision-k8s-1.24/1561265652461735936